docs/poly-apis.md

```diff
@@ -0,0 +1,636 @@
+
+#### Example
+
+|  JSON | Binary |
+| --- |--- |
+|  “0x” | Empty binary |
+|  “0x00” | Single byte 0 |
+|  “0x636b62” | 3 bytes, UTF-8 encoding of ckb |
+|  “00” | Invalid, 0x is required |
+|  “0x0” | Invalid, each byte requires 2 digits |
+
+### Type `Versions`
+
+Version info of Godwoken & Web3 & Web3 Indexer.
+
+#### Examples
+
+
+```
+{
+  "web3Version": "1.0.0-rc1",
+  "web3IndexerVersion": "1.0.0-rc1",
+  "godwokenVersion": "1.1.0 f3cdd47"
+}
+```
+
+
+#### Fields
+
+`Versions` is a JSON object with the following fields.
+
+*   `web3Version`: `string` - Version of Godwoken Web3
+
+*   `web3IndexerVersion`: `string` - Version of Godwoken Web3 Indexer
+
+*   `godwokenVersion`: `string` - Version of Godwoken
+
+
+### Type `RollupCell`
+
+#### Fields
+
+`RollupCell` is a JSON object with the following fields.
+
+*   `typeHash`: [`H256`](#type-h256) - Hash of typeScript
+
+*   `typeScript`: [`Script`](#type-script) - Deploy info of rollup cell
+
+
+### Type `RollupConfig`
+
+#### Fields
+
+`RollupConfig` is a JSON object with the following fields.
+
+*   `requiredStakingCapacity`: [`Uint64`](#type-uint64) - Stacking capacity
+
+*   `challengeMaturityBlocks`: [`Uint64`](#type-uint64) - Challenge maturity blocks
+
+*   `finalityBlocks`: [`Uint64`](#type-uint64) - Finality Blocks
+
+*   `chainId`: [`Uint64`](#type-uint64) - Chain ID
+
+
+
+### Type `NodeInfo`
+
+Info of Godwoken & Web3 node.
+
+#### Fields
+
+`NodeInfo` is a JSON object with the following fields.
+
+*   `nodeMode`: `fullnode | readonly` - fullnode or readonly node
+
+*   `rollupCell`: [`RollupCell`](#type-rollupcell) - Rollup cell info
+
+*   `rollupConfig`: [`RollupConfig`](#type-rollupconfig) - Rollup config info
+
+*   `gwScripts`: [`GwScripts`](#type-gwscripts) - Godwoken scripts deploy info
+
+*   `eoaScripts`: [`EoaScripts`](#type-eoaScripts) - Supported EOA scripts
+
+*   `backends`: [`Backends`](#type-backends)
+
+*   `accounts`: [`Accounts`](#type-accounts) 
+
+*   `web3ChainId`: [`Uint64`](#type-uint64) - Chain ID of web3
+
+
+### Type `GwScripts`
+
+Godwoken scripts deploy info
+
+#### Fields
+
+`GwScripts` is a JSON object with the following fields.
+
+*   `deposit`: [`ScriptInfo`](#type-scriptinfo) - Deposit script
+
+*   `withdraw`: [`ScriptInfo`](#type-scriptinfo) - Withdrawal script
+
+*   `stateValidator`: [`ScriptInfo`](#type-scriptinfo) - State validator script
+
+*   `stakeLock`: [`ScriptInfo`](#type-scriptinfo) - State lock script
+
+*   `custodianLock`: [`ScriptInfo`](#type-scriptinfo) - Custodian lock script
+
+*   `challengeLock`: [`ScriptInfo`](#type-scriptinfo) - Challenge lock script
+
+*   `l1Sudt`: [`ScriptInfo`](#type-scriptinfo) - L1 sudt script
+
+*   `l2Sudt`: [`ScriptInfo`](#type-scriptinfo) - L2 sudt script
+
+*   `omniLock`: [`ScriptInfo`](#type-scriptinfo) - Omni lock script
+
+*   `challengeLock`: [`ScriptInfo`](#type-scriptinfo) - Challenge lock script
+
+
+### Type `EoaScripts`
+
+Supported eoa scripts
```

Comment:
```suggestion
Supported eoa account lock scripts
```

Reply:
* I think "EOA" is better to be uppercase.
* s/Supported lock scripts for EOA accounts/

Reply:
Updated